cmd/notation/verify.go

```diff
@@ -137,11 +138,11 @@ func appendCertPathFromName(paths, names []string) ([]string, error) {
 		if err != nil {
 			return nil, err
 		}
-		path, ok := cfg.VerificationCertificates.Certificates.Get(name)
-		if !ok {
+		idx := slices.Index(cfg.VerificationCertificates.Certificates, name)
+		if idx < 0 {
 			return nil, errors.New("verification certificate not found: " + name)
 		}
-		paths = append(paths, path)
+		paths = append(paths, cfg.VerificationCertificates.Certificates[idx].Path)
 	}
 	return paths, nil
```

Comment:
fyi, VerificationCertificates though config.json will go away when we implement updated [trust store](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-store) and [trust policy](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-store) spec.